bricks/aem-testimonials/aem-testimonials.html

```diff
@@ -0,0 +1,162 @@
+      flex: 1;
+      text-align: center;
+      cursor: pointer;
+      width: 100%;
+      position: relative;
+      color: var(--tabs-text-color);
+      background: transparent;
+      padding: 16px 0 8px;
+      font-family: var(--body-font-family);
+      font-weight: 600;
+      border: none;
+      border-bottom: 2px solid #e8e8e8;
+      transition: all 0.2s ease-in-out;
+      border-radius: 0;
+      height: 92px;
+      opacity: 0.35;
+      transition: opacity 0.2s ease-in-out;
+    }
+
+    .aem-testimonials__tabs__button[aria-selected='true'] {
+      border-bottom-color: #000;
+      opacity: 1;
+    }
+
+    .aem-testimonials__tabs__button:hover {
+      opacity: 1;
+    }
+
+    .aem-testimonials__tabs__button img {
+      width: 70%;
+      height: 100%;
+      max-height: 56px;
+      object-fit: contain;
+      pointer-events: none;
+    }
+
+    .aem-testimonails__content {
+      display: flex;
+      flex-wrap: wrap;
+      gap: 30px;
+      align-items: center;
+      margin-top: var(--spacing-s);
+    }
+
+    .aem-testimonails__content__image {
+      width: 100%;
+    }
+
+    @media screen and (min-width: 768px) {
+      .aem-testimonails__content__image {
+        width: 50%;
+      }
+    }
+
+    .aem-testimonails__content__image img {
+      width: 100%;
+      height: auto;
+    }
+
+    .aem-testimonails__content__text {
+      flex: 1;
+    }
+
+    .aem-testimonials__content__quote {
+      font-weight: 700;
+      font-size: var(--type-body-s-size);
+      line-height: 1.25;
+      text-align: left;
+      padding-bottom: var(--spacing-s);
+      margin: 0;
+    }
+
+    .aem-testimonials__content__quote p {
+      margin: 0;
+    }
+
+    .aem-testimonials__content__author {
+      display: grid;
+      width: 100%;
+      grid-template-columns: 50px 1fr auto;
+      align-items: center;
+      padding-bottom: var(--spacing-s);
+      grid-gap: var(--spacing-xs);
+    }
+
+    .aem-testimonials__content__author__avatar img {
+      border-radius: 50%;
+      overflow: hidden;
+    }
+
+    /* Name */
+    .aem-testimonials__content__author__name p {
+      font-weight: 700;
+      font-size: 0.83em;
+      margin: 0 0 0.25em;
+      line-height: 1.2;
+    }
+
+    .aem-testimonials__content__author__name p + p {
+      font-weight: 400;
+      font-style: normal;
+      font-size: var(--type-body-xxs-size);
+      margin-bottom: 0;
+    }
+  </style>
+
+  <aem-tabs active="0">
+    <div class="aem-testimonials__tabs">
+      <button
+        class="aem-testimonials__tabs__button"
+        data-tabs-label
+        data-aem-each=":scope > div > div:nth-child(1)"
+      >
+        <div data-aem-inject="picture"></div>
+      </button>
+    </div>
+
+    <div
+      class="aem-testimonails__content"
+      data-tabs-content
+      data-aem-each=":scope > div"
+    >
+      <div
+        class="aem-testimonails__content__image"
+        data-aem-inject=":scope > div:nth-child(2)"
```

Comment:
Note: the `:scope` of items inside `data-aem-each` are each item iterate through thus each internal query are reduce to the HTMLFragment rather than the full doc.